src/main.rs

```diff
@@ -1,32 +1,39 @@
-use std::env;
-use std::fs;
+#![feature(seek_stream_len)]
+#![warn(clippy::pedantic)]
```

Comment:
this enables pedantic lints when you run `cargo clippy`. It's not a bad way to learn Rust idioms (but as the name suggests, it can be quite pedantic!)